src/components/include/security_manager/ssl_context.h

```diff
@@ -103,6 +104,14 @@ class SSLContext {
                        size_t* out_data_size) = 0;
   virtual bool IsInitCompleted() const = 0;
   virtual bool IsHandshakePending() const = 0;
+  /**
+  * @brief GetCertificateDueDate gets certificate expiration date
+  * @param due_date - certificate expiration time to be received
+  * @return True if certificate expiration date receivedssss
```

Comment:
@AByzhynar receivedssss - typo

Reply:
@LitvinenkoIra Fixed in 319bbf54b59d3e4e5cd7f5f46d461895fa877c9d